passwords.md

```diff
@@ -110,12 +110,13 @@ Of course, we need to define a route to actually handle the password reset form
     use Illuminate\Support\Facades\Hash;
     use Illuminate\Support\Facades\Password;
     use Illuminate\Support\Str;
+    use Illuminate\Validation\Rules\Password;
```

Comment:
I know it's documentation, but, this clashes with the facade import above, people will complain lol

Reply:
Ups, nice catch. One second.

Reply:
fixed